experimental/FTheoryTools/test/literature_models.jl

```diff
@@ -367,3 +367,37 @@ h = literature_model(arxiv_id = "1507.05954", equation = "3.4")
   @test is_partially_resolved(h) == false
   @test string.(zero_section(h)) == ["0", "-b1", "a1"]
 end
+
+
+
+#############################################################################
+# 8: Test models from F-theory on all toric hypersurfaces over arbitrary base
+#############################################################################
+
+foah1 = literature_model(arxiv_id = "1408.4808v2", equation = "3.4")
+
+@testset "Test defining data for literature model defined by model index" begin
+  @test dim(base_space(foah1)) == 3
+  @test dim(ambient_space(foah1)) == 5
+  @test is_base_space_fully_specified(foah1) == false
+  @test model_description(foah1) == "F-theory hypersurface model with fiber ambient space F_1"
+end
+
+
+
+#############################################################################
+# 9: Test models from F-theory on all toric hypersurfaces over concrete base
+#############################################################################
+
+B3 = projective_space(NormalToricVariety, 3)
+Kbar = anticanonical_divisor(B3)
+foah1_B3 = literature_model(arxiv_id = "1408.4808v2", equation = "3.4", base_space = B3, model_sections = Dict("s7" => Kbar, "s9" => Kbar), completeness_check = false)
+
+@testset "Test defining data for literature model defined by model index" begin
+  @test dim(base_space(foah1)) == 3
+  @test dim(ambient_space(foah1)) == 5
+  @test is_base_space_fully_specified(foah1) == true
+  @test model_description(foah1) == "F-theory hypersurface model with fiber ambient space F_1"
```

Comment:
```suggestion
  @test dim(base_space(foah1_B3)) == 3
  @test dim(ambient_space(foah1_B3)) == 5
  @test is_base_space_fully_specified(foah1_B3) == true
  @test model_description(foah1_B3) == "F-theory hypersurface model with fiber ambient space F_1"
```

(My mistake earlier, but did not want to push again to cause merge conflicts for Mikelis.)

Reply:
This mistake of mine causes some (if not all) of the test failures.